packages/shared/src/vis-models.ts

```diff
@@ -66,6 +66,15 @@ export interface Dims {
 
 export type IgnoreValue = (val: number) => boolean;
 
+export interface ExportEntry {
+  format: string;
+  url: ExportURL;
```

Comment:
`url` used to be option; now it isn't.

Reply:
`ExportEntry` was already defined in `ExportMenu` before. It's meant to type an entry in the export menu, so I think the naming is good as is.

`BuiltInExporter` is the term I used for functions that generate the export payload in the mapped components. The provider then generates an `ExportURL` async function that invokes this exporter, and the mapped components use those `ExportURL`s in the export entries they pass to the toolbars.